src/sentry/event_manager.py

```diff
@@ -2259,6 +2285,17 @@ def _get_priority_for_group(severity: Mapping[str, Any], kwargs: Mapping[str, An
         return PriorityLevel.MEDIUM
 
 
+def update_severity_error_count(reset=False) -> None:
+    if reset:
+        cache.set(SEER_ERROR_COUNT_KEY, 0, timeout=60 * 60)
+        return
+
+    try:
+        cache.incr(SEER_ERROR_COUNT_KEY)
```

Comment:
We can use `cache.touch(key, time=timeout)` to bump the ttl here

https://docs.djangoproject.com/en/5.0/topics/cache/#basic-usage